11ty/index.njk

```diff
@@ -15,9 +15,24 @@ layout: layouts/index.njk
       With the foundation of vocabulary, we can begin to understand lived experiences of people different than us. Words can provide us with a sense of identity and allow us to find kinship through common experiences.
     </p>
   </div>
-  <div class="box">
-    <h2 class="sub-headline">Ways to help</h2>
-    <ol class="list">
+  <div
+    class="box spoiler spoiler_collapsed"
+    data-spoiler-mobile-only="true"
+    id="ways-to-help">
+    <h2 class="sub-headline">Ways to help
+      <button
+        aria-expanded="true"
+        aria-controls="ways-to-help-content"
+        class="spoiler__toggle-btn"
+        id="ways-to-help-visibility-toggle">
+        {# https://css.gg/chevron-down #}
+        <svg xmlns="http://www.w3.org/2000/svg" width="24" height="24" fill="none" viewBox="0 0 24 24"><path fill="currentColor" d="M6.34 7.76L4.93 9.17 12 16.24l7.07-7.07-1.41-1.41L12 13.4 6.34 7.76z"/></svg>
+      </button>
+    </h2>
+    <ol
+      class="list spoiler__content"
+      id="ways-to-help-content"
+      aria-labelledby="ways-to-help-visibility-toggle">
```

Comment:
Blocking: This `aria-labelledby` does not work, as it’s pointing to the empty button. If we add toggle text to the button, this is probably not what we want as a label for the list. I’d propose to point to the headline, and move the button out of the headline (e.g. having the button and headline in a `header`. 

```suggestion
      aria-labelledby="ways-to-help"
```

Reply:
Fixed in c73385a